documentation/modules/proc_migrating-virtual-machines-cli.adoc

```diff
@@ -627,10 +627,22 @@ spec:
     destination:
       name: <destination_provider>
       namespace: <namespace>
+  offloadOptions:
+    offloadPlugin: <offload_plugin> <3>
+    storageSecret:
+      name: <storage_secret_name> <4>
+      namespace: <namespace>
+    storageVendorProduct: <storage_product> <5>
 EOF
 ----
 <1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
```

Comment:
worth mentioning that accessMode is optional

Reply:
yes it is optional

Reply:
@rgolangh Changed to <1> Optional for storage copy offload, required for other VMware migrations: Allowed values are `ReadWriteOnce` and `ReadWriteMany`.